src/pybind/mgr/influx/module.py

```diff
@@ -0,0 +1,211 @@
+import json
+import sys
+import time
+from ConfigParser import SafeConfigParser
+from collections import defaultdict
+from influxdb import InfluxDBClient
+from mgr_module import MgrModule
+from datetime import datetime
+from threading import Event 
+
+class Module(MgrModule):
+ 
+    run = True 
```

Comment:
`run` should be an instance attribute defined in __init__, rather than a class attribute